### PR TITLE
Revise user/quick-start/installation/index

### PR DESCRIPTION
**Pull request by davidjharder**
- Add steps for verifying ISO on Windows
- Rearrange the "Verifying" sections generally and simplify some wording
- Harmonize heading levels throughout the page
- Remove some horizontal seperators, we don't use those anymore

Fixes https://github.com/getsolus/help-center-docs/issues/420

### CODE REVIEW

File: docs/user/quick-start/installation/index.md [hunk 1]
```diff
@@ -5,57 +5,75 @@ summary: Installation Guide for multiple operating systems
...
 ```
 
----
+### Windows
+
+#### Verifying signed SHA256SUM file
+
+On Windows, first install [gpg4win](https://www.gpg4win.org/).
+
+Now you can run the same `gpg` commands from the the _Verify SHA256SUMS file signature_ section above.
```
Comment (EbonJaeger) [inline docs/user/quick-start/installation/index.md:51]: Do we want to link to the section above? Saves the reader from scrolling to find it. I can never remember the syntax for a link like that, though.

```markdown
[Verify SHA256SUMS file signature](#Verifying-the-SHA256SUMS-file-signature)
```

Maybe? You'd have to play around with it.

**Review by EbonJaeger — COMMENTED** [review]
Just one question, otherwise LGTM.

File: docs/user/quick-start/installation/index.md (hunk 1 above)
Comment (davidjharder) [reply docs/user/quick-start/installation/index.md]: Maybe there's a syntax for links within the same file. Too lazy to find out, use absolute path instead

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks! :)

**Pull request merged**